packages/design-system-react/src/components/AvatarFavicon/AvatarFavicon.tsx

```diff
@@ -1,14 +1,11 @@
-import React, { useState } from 'react';
+import React, { useState, forwardRef } from 'react';
```

Comment:
Fixed in https://github.com/MetaMask/metamask-design-system/pull/673 so will be removed after rebase